docs/DeveloperGuide.md

```diff
@@ -333,47 +333,47 @@ The following activity diagram summarizes what happens when a user interacts wit
+| `***` | Frequent User          | I can undo my actions                                                        | I can correct my mistakes                                                                        | 
+| `***` | Expert User            | Quickly update the status of multiple elderly contacts                       | I can efficiently manage my tasks                                                                | 
+| `***` | New user               | Try out the app with sample data                                             | I can familiarise myself with it without worrying about the data I am playing around with        | 
+| `***` | Onboarded user         | Purge all sample data                                                        | The app is ready to be used                                                                      | 
+| `***` | Frequent User          | I can view a history of calls made to an elderly contact                     | I can track past interactions                                                                    | 
+| `***` | Frequent User          | Change the frequency of calls needed to be made for each elderly             | I can track when I need to make calls                                                            | 
+| `***` | Expert User            | Archive elderly contacts who are temporarily not part of the program         | I can keep my contact list organised                                                             | 
+| `***` | Frequent User          | Export/Import my data across computers                                       | If I would like to work using separate computers, e.g. a laptop and desktop, I can keep them updated | 
+| `***` | Expert User            | Set multiple contacts as emergency contacts                                  | I can call their next of kin in the event the elderly do not pick up or needs help               | 
+| `***` | Frequent User          | Export a specific elderly's details and history (All information)            | So I can share it with another employee                                                          | 
+ | `**` | Frequent User          | Make changes to the style of my address book                                 | I can be happy when I use the address book                                                       | 
+ | `**` | Frequent user          | Mark contacted elderly quickly using shortcuts                               | I can work efficiently to contact the entire list                                                | 
+ | `**` | Detail-orientated user | Add tags to each elderly                                                     | I can keep track of details that are important when taking the call.                             | 
```

Comment:
Should we remove the fullstop here for consistency or add fullstop for other rows also, up to you.